index.html

```diff
@@ -5,19 +5,22 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;400;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="css/style.css">
     <title>Document</title>
 </head>
 
 <body>
     <header>
         <a href="#home">Eyasu</a>
+
+
         <nav>
-            <img class="nav-pic" src="./img/Union.png" alt="">
-            <a href="#portifolio">Home</a>
-            <a href="#about">About</a>
-            <a href="#contact">Contact</a>
+            <img class="nav-pic" src="img/union.png" alt="">

```

Comment:
[ OPTIONAL BUT RECOMMENDED] Hamburger does not show on mobile view. Kindly make sure that there is a hamburger menu for navigation on the right side of the header.

![hamburger](https://user-images.githubusercontent.com/35504149/171667328-fcf15329-dd30-4cc9-b744-06ba31707791.png)